src/Chainweb/SPV/CreateProof.hs

```diff
@@ -259,6 +260,27 @@ outputProofPrefix i db payload = do
   where
     blockOutputTable = _payloadCacheBlockOutputs $ _payloadCache db
 
+-- TODO: incorporate version into crossnet: syntax?
```

Comment:
I think this could be a good idea, but maybe @sirlensalot has a better intuition.